debugging_protected_audience_api_services.md

```diff
@@ -200,6 +200,43 @@ The following errors will be available for tracking:
     partitioned by destination:** Similar to the above, but partitioned by the
     request destination.
 
+## Debugging data
+Bidding and Auction servers can be built in 2 [build_flavor](https://github.com/privacysandbox/bidding-auction-servers/blob/e40a4fccdce168379189ab7b6b87b55b1e3f736d/BUILD#L163), `prod` and `non_prod`, only `prod` image can pass attestation and decrypt
+production request, but `non_prod` have more debugging methods with requests with fake encryption.
+
+1. Verbose logs
+
+    This is the verbose logs from the B&A code.
+    - Export channel
+      - In `prod`, they are only exported through Open Telemetry log, which can be viewed in cloud log explorer.
+      - In `non_prod`, they are also printed in stderr.
+
+    - Log types
+      - System logs:
+        These logs are not related to user device request, they are always exported in both `prod` and `non_prod`.
+      - Request logs:
+        These logs are the result of server processing encrypted request. With `prod`, they are only exported for [AdTech consented debugging](#adtech-consented-debugging). With `non_prod`, they are always exported.
+
+    Each server has `PS_VERBOSITY` flag, which sets the verbosity level. High `PS_VERBOSITY` can slow down the `non_prod` server, because there can be large volume of request logs at high QPS, `PS_VERBOSITY` should be `<=3` in this case.
```

Comment:
```suggestion
    Each server has `PS_VERBOSITY` flag, which sets the verbosity level of the output logs. A higher verbosity will give more logging data, but can also slow down request processing. This is especially relevant in `non_prod` builds because there can be large volume of request logs. At high QPS, we recommend setting `PS_VERBOSITY`   `<=3`.
```